### DISCUSSION TITLE
Let's leave Lotus-Market and develop it.

### DISCUSSION THREAD

**Discussion by Shekelme** [filecoin-project/lotus#10557] [Ideas]
Hi all!
I understand the utter futility of the question I am raising here, but I still want to try to do it. At least in order to say to myself, "I did everything I could," in any case.
Let's leave the Lotus-Market and develop it again. So that the Lotus software was a complete product, all parts of which were developed from the first line of code for mutual coordinated work.
As a participant in the Filecoin network from the very beginning of the testnet race, I never understood why it was necessary to put aside all the work that was invested in the Lotus-Market in order to debug the problems that were constantly detected in `boost` for last several months.
As an SP, I really want to have as little body movements as possible and as much automatism as possible. So far, it seems to me personally that `boost` adds more problems than convenience. Is it really so necessary? Please change your mind. Perhaps I am not the only one who thinks so, but the others are just shy to raise this issue or do not want to waste their time on it.

p.s. if you have nothing to say on it in a constructive way, please just skip post.

**Comment by brendalee** [comment]
@Shekelme thank you for the input! It would be great to learn more about some of the issues that you had with Boost, and what type of end to end experience you would prefer when working with Lotus+Boost together.
